ts/src/crs/index.ts

```diff
@@ -0,0 +1,175 @@
+import { readFile } from "fs/promises";
```

Comment:
I think I see this file in a3-circuit/ts too. Does it belong in both?

Reply:
It belongs here. I will remove the redundancy soon.